spec/features/users/show_spec.rb

```diff
@@ -44,7 +44,7 @@
 
           click_on "My Orders"
 
-          expect(current_path).to eq(profile_orders_path(@user))
+          expect(current_path).to eq(profile_orders_path)
```

Comment:
Also redoing the Route allowed for us to not have to pass in the user, as it uses current_user. Should be fine for now, we'll see how it works with Admin @WHomer